crocket/src/main/java/com/crocket/CroquetView.java

```diff
@@ -1,32 +1,71 @@
 package com.crocket;
 import javax.swing.*;
 
+import java.awt.Component;
 import java.awt.Dimension;
 import java.awt.Toolkit;
 
+/*
+ * Class: CroquetView
+ * 
+ * This class extends JFrame and represents the main window of the game. It is responsible for 
+ * rendering the game state to the user.
+ * 
+ * CroquetView is implemented as a Singleton, ensuring that only one instance of CroquetView exists
+ * in the application at any time. This is done to maintain a consistent view state throughout
+ * the application.
+ * 
+ * Methods:
+ * - drawLevel(LevelView drawLevel): Adds the provided LevelView to the JFrame, effectively drawing the level.
+ * - removeAllLevelComponents(): Removes all LevelView components from the JFrame.
+ * - getInstance(): Returns the singleton instance of CroquetView.
+ * 
+ */
+
 public class CroquetView extends JFrame{
-    JFrame frame;
-    Dimension screen = Toolkit.getDefaultToolkit().getScreenSize();
-    DrawEntity entity = new DrawEntity();
-    DrawField field = new DrawField();
-    ImageIcon img = new ImageIcon("crocket/assets/textures/JFrame_Icon.jpg");
-    String title = "Krocket";
-    boolean visibility = true;
+    private static CroquetView INSTANCE;
```

Comment:
All caps names are used for constants in Java naming conventions. Maybe change to `instance`?

Reply:
Changed INSTANCE to instance